README.md

```diff
@@ -114,14 +114,17 @@ There are several approaches of addressing the pile-up in the detector:
   It creates histograms filling in the information on the deposits per event. This can be later scaled with *sqrt(mu)* for the noise (RMS of the energy distributions) and with *mu* for the mean energy deposit.
   Detailed analysis and this scaling is done with FCC_calo_analysis_cpp toolkit.
 
-- apply estimated noise levels at the cluster level for sliding window reconstruction or layer by layer for the topological clusters (**--addPileupNoise**).
+- apply estimated noise levels at the cluster level for sliding window reconstruction or per cell for the topological clusters (**--addPileupNoise**).
 
 2. Mix already simulated events in order to overlay:
 
-- cells that are later passed to the reconstruction (**--mergePileup**)
 
-- or clusters that can be directly analysed (not yet supported)
+2.1. merge MinBias events (**--mergePileup**)
 
+2.2. merge signal and PU events that are later passed to the reconstruction (**--addPileupToSignal**)
```

Comment:
yes, and i tried to produce them on the fly, but this took extemely long, such that the jobs would be killed.. so i added the intermediate step. For the new config file, this is way more easy, then trying to include this in the  overlayPileup.py becasue the input is passed differently. you only have to define the process of your signal and the pu events are added. The input of the overlayPileup.py is the primary process itself, so it would not be trivial to add the signal in a consistent way.

Reply:
So you are saying that you duplicated the whole configuration just because the flags `--inName` and `--inSignalName` are refering to pileup and signal, respectively? (in `overlayPileup.py`)
Why don't you rename them to `inPileupName` and `inName`? The pileup needs to be passed anyway in a special format as the [input](https://github.com/FCC-hh-framework/FCCSimJobs/blob/master/python/send.py#L528). This way you can have signal event passed as usual. (However make sure that mergePileup still works as expected)